Replace sys/unistd.h with unistd.h

There is no sys/unistd.h on Android. Including <unistd.h> instead does the
trick.
Index: libdaemon/libdaemon-0.14/examples/testd.c
===================================================================
--- libdaemon.orig/libdaemon-0.14/examples/testd.c
+++ libdaemon/libdaemon-0.14/examples/testd.c
@@ -23,7 +23,7 @@
 #include <string.h>
 #include <sys/types.h>
 #include <sys/time.h>
-#include <sys/unistd.h>
+#include <unistd.h>
 #include <sys/select.h>
 
 #include <libdaemon/dfork.h>
